maps/~map_system/maps.dm

```diff
@@ -59,6 +59,7 @@ var/list/all_maps = list()
 	var/list/holomap_legend_x = list()
 	var/list/holomap_legend_y = list()
 	var/list/meteor_strike_areas	// VOREStation Edit - Areas meteor strikes may choose to hit.
+	var/list/drop_pod_allowed_z		// VOREStation Edit - Which z-levels to target with droppods?
```

Comment:
@RadiantFlash7 And it's use here.

Reply:
I... I don’t even, Why is that a var they added?
Droppods are admin spawn only, unless they added a syndicate droppod-transport?

Reply:
See Devchat. :D